src/Integration.Vsix/CFamily/CFamilyIssueToAnalysisIssueConverter.cs

```diff
@@ -139,7 +140,7 @@ private AnalysisIssueLocation ToAnalysisIssueLocation(MessagePart cFamilyIssueLo
         {
             return new AnalysisIssueLocation
             (
-                filePath: cFamilyIssueLocation.Filename,
+                filePath: cFamilyIssueLocation.Filename.Replace('/', '\\'),
```

Comment:
The best way to get a path in canonical form is to call `Path.GetFullPath...)`.
The path/file doesn't have to exist, and the method will fix this issue and others e.g. duplicate directory separators and paths containing `.` and `..`.

Reply:
That's what I started with originally. However, if the file path is absolute, it will be placed under the current directory. I was afraid it would lead to some side effects. WDYT?

Reply:
That's worth commenting in the code, because it isn't obvious that's why it's done like that.

Does the `subprocess.exe` ever return relative paths?

Reply:
At the moment - no, it shouldn't. 

On second thought, I'll change it to `Path.GetFullPath`: even if the subprocess returns a relative path, we will fail to match the issues. So the behavior would be the same between a relative path and a wrong absolute one.